src/app/GitUI/Editor/Diff/DiffViewerLineNumberControl.cs

```diff
@@ -139,6 +146,8 @@ public void Clear()
         MaxLineNumber = 0;
     }
 
+    public override bool IsVisible => _visible;
+
```

Comment:
What is your plan with this unused property?

Reply:
Used in SelectedLineChanged.SelectedLineChanged()